version/info.go

```diff
@@ -32,3 +34,16 @@ var Map = map[string]string{
 	"buildDate": BuildDate,
 	"goVersion": GoVersion,
 }
+
+func init() {
+	buildInfo := prometheus.NewGaugeVec(
```

Comment:
Just for the record: With the plans I have for `client_golang`, registering a metric with a constant value will be one line (because of auto-registering). `NewConstMetric` is really meant for something to throw away and explicitly not to register.